src/data/index.js

```diff
@@ -1820,4 +1820,52 @@ export default [
       'Bloomberg',
     ],
   },
+  {
+    id: 164,
+    name: 'Longest Word in Dictionary',
+    url: 'https://leetcode.com/problems/longest-word-in-dictionary/',
+    pattern: 'Trie',
+    difficulty: 'Easy',
+    companies: [],
+  },
+  {
+    id: 165,
+    name: 'Maximum XOR of Two Numbers in an Array',
+    url: 'https://leetcode.com/problems/maximum-xor-of-two-numbers-in-an-array',
+    pattern: 'Trie',
+    difficulty: 'Medium',
+    companies: [],
```

Comment:
The companies for `'Maximum XOR of Two Numbers in an Array'` is `'Google'`

![image](https://user-images.githubusercontent.com/13009507/82843185-5a030e00-9eaa-11ea-9e63-91b26f20601a.png)